Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataParametersFactory.cs

```diff
@@ -45,8 +45,7 @@ public IDictionary<string, Parameter> Create(bool isCompositeContext)
                         format = "int32",
                         minimum = 0,
                         maximum  = _defaultPageSizeLimitProvider.GetDefaultPageSizeLimit(),
-                        required = false,
-                        @default = 25
```

Comment:
I this change acceptable? I'm removing the default `limit` of 25 in SwaggerUI, which will make the API use the default limit defined in appSettings.json (currently 500).
This change is so that users don't have to manually remove the limit when using key set paging.

Reply:
@axelmarquezh -- Per discussion with @vimayya , we should continue to provide default value in Swagger UI of 25 for `limit` and (now) `pageSize`. At the API level, we will only raise a validation error if both `offset` and `pageToken` are provided (which are mutually exclusive options), and we'll be tolerant of `limit` and `pageSize` being provided together.

Reply:
Brought back the default `limit` of 25 and also set a default `pageSize` of 25.